src/mcts/node.h

```diff
@@ -220,15 +226,12 @@ class Node {
   ConstIterator Edges() const;
   Iterator Edges();
 
-  class NodeRange;
-  // Returns range for iterating over nodes. Note that there may be edges
-  // without nodes, which will be skipped by this iteration.
-  NodeRange ChildNodes() const;
```

Comment:
Just documenting some history that this was added in #145 and the last use was removed in #251.

And apparently imaginary friends are okay 😉 
https://github.com/LeelaChessZero/lc0/blob/b4f1dbc60a2befe4e56384441ac4a91e2972354b/src/mcts/node.h#L313-L314
(`Node_Iterator` is gone and `Edge` doesn't actually need to be a friend.)

Reply:
Good point about Node_Iterator. Fixed.